exp/views/study.py

```diff
@@ -669,6 +589,161 @@ def build_researchers_paginator(self, researchers_result):
         return self.paginated_queryset(researchers_result, page, 10)
 
 
+class ManageResearcherPermissionsView(StudyDetailView):
```

Comment:
I *think* it may be simpler not to extend StudyDetailView for these, but I'm not sure - it seems like there's a bunch of functionality in the StudyDetailView that we're not using in any of these POST views, and it might just be possible to separate out what belongs where.